ops/services/web_application_firewall/main.tf

```diff
@@ -78,10 +86,33 @@ resource "azurerm_web_application_firewall_policy" "sr_waf_policy" {
       selector_match_operator = "Equals"
     }
 
+    exclusion {
+      match_variable          = "RequestArgNames"
+      selector                = "iss"
+      selector_match_operator = "Equals"
+    }
+    exclusion {
+      match_variable          = "RequestArgNames"
+      selector                = "street" //Will need to turn on 942440 and 942110 if not effective.
+      selector_match_operator = "Equals"
+    }
+    exclusion {
+      match_variable          = "RequestArgNames"
+      selector                = "namePrefixMatch" //Will need to turn on 942330 if not effective
```

Comment:
Very uninformed questions ahead:

Are these selectors matching on GraphQL queries as well? Or are we sending `namePrefixMatch` as a request param somewhere?

Also, how come we need a manual exception for `namePrefixMatch` specifically? Does the selector name match something on a WAF blocklist?

Reply:
All good! The more questions, the better, if you ask me.

Yes, these selectors are matching on GraphQL queries. This one was triggered by a name such as `O'Connor` coming across in a query on the `/graphql` API endpoint. 

In this case, the way our query data is formatted, the WAF sees the `O` and apostrophe as a SQL quotation, and interprets it as a SQL injection attempt. This is a known issue with our site, and rather than turn off injection detection altogether, I just want to turn it off for this specific argument, as I know it will be problematic. The downside to doing that is that it will cause the WAF to stop evaluating on all GraphQL queries that have `namePrefixMatch` as an argument...so, this may actually do more harm than turning off the evaluation of the rule.

Now that I've typed that all out...I think I actually just want to turn that rule off. I'll modify this to match. Thank you for making me think that over!